antlr/src/main/antlr4/org/apache/iotdb/db/qp/strategy/SqlBase.g4

```diff
@@ -1272,6 +1268,8 @@ UNDERLINE : '_';
 
 NaN : 'NaN';
 
+SDT : 'SDT';
```

Comment:
Why write like this. Does SDT didn't need case insensitive?

Reply:
removed. thank you!